net-misc/i2pd/i2pd-2.6.0-r3.ebuild

```diff
@@ -53,27 +53,38 @@ src_configure() {
 
 src_install() {
 	cmake-utils_src_install
+
+	# config
+	insinto "/etc/i2pd"
+	doins "${S}/docs/i2pd.conf"
+	doins "${S}/debian/subscriptions.txt"
+	doins "${S}/debian/tunnels.conf"
+
+	# doc
 	dodoc README.md
+
+	# working directory
 	keepdir /var/lib/i2pd/
-	insinto "/var/lib/i2pd"
+	insinto /var/lib/i2pd
```

Comment:
Removing the quotes is fine, except that its inconsistent with what you do on line 58.